readthedocs/projects/tasks.py

```diff
@@ -249,11 +249,11 @@ def sync_versions(self, version_repo):
         version_post_data = {'repo': version_repo.repo_url}
         tags = None
         branches = None
-        if all([
-                version_repo.supports_lsremote,
-                not version_repo.repo_exists(),
-                self.project.has_feature(Feature.VCS_REMOTE_LISTING),
-        ]):
+        if (
+            version_repo.supports_lsremote and
+            not version_repo.repo_exists() and
+            self.project.has_feature(Feature.VCS_REMOTE_LISTING)
```

Comment:
We want to shor-circuit here, so we only execute the command if supports_lsremote is true.